SmartHomeManager/SmartHomeManager.DataSource/AccountDataSource/AccountRepository.cs

```diff
@@ -28,14 +28,17 @@ public async Task<bool> AddAsync(Account account)
             return true;
         }
 
-        public Task<bool> DeleteAsync(Account account)
+        public bool Update(Account account)
         {
-            throw new NotImplementedException();
+            _dbContext.ChangeTracker.Clear();
```

Comment:
Just curious why is this called here, this clears all pending changes being tracked by the dbContext.